.buildkite/pipeline.yml

```diff
@@ -74,6 +74,22 @@ steps:
     commands:
     - nix-build ci.nix -A haddock --no-out-link
 
+  - label: MANUAL scheduled delphinet test
```

Comment:
IIUC you have added this here to test the changes yourself only and should be removed before merging, am I correct?

Reply:
Yeah, I'll drop this change when the tests succeed and the MR is ready to be merged.